api/proto/dop/devflowrule/devflowrule.proto

```diff
@@ -82,31 +83,45 @@ message GetDevFlowRuleResponse {
 message DevFlowRule {
     string ID = 1 [json_name = "id"];
     repeated Flow flows = 2 ;
-    uint64 orgID = 3;
-    string orgName = 4;
-    uint64 projectID = 5;
-    string projectName = 6;
-    google.protobuf.Timestamp timeCreated = 7;
-    google.protobuf.Timestamp timeUpdated = 8;
-    string creator = 9;
-    string updater = 10;
+    repeated BranchPolicy branchPolicies = 3;
```

Comment:
for proto, you should add new field to the end

Reply:
done